### DISCUSSION TITLE
Fraud via github

### DISCUSSION THREAD

**Discussion by RedProGroup** [community/community#128297] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Good day!
I want to tell you about the scammer https://github.com/prometeydev, his project https://github.com/prometeydev/Prometheus. Also, this project is infected, thereby spreading viruses.
I wanted to buy the Prometheus VIP product from him and paid him 60 USDT, after which he deleted part of the history and then completely deleted his avatar from the Telegram account. I attach part of the correspondence and payment receipts in the archive; the ChatExport_2024-06-12 folder is an exported chat from Telegram.
When you run his program in Roaming, the required.exe file appears and is registered in run. Also, malicious files main.exe, setup.exe and svchost.exe, etc. are created in ProgramData. svchost.exe is an unkillable process, you can only delete it through unlocker-1-9-2, at least I deleted it through unlocker-1-9-2.
Take action against the scammer https://github.com/prometeydev.

Sincerely, RedProGroup
[Prometheus scammer.zip](https://github.com/user-attachments/files/15811066/Prometheus.scammer.zip)

**Comment by billythekid** [comment]
Hi @RedProGroup :wave:

Please use this functionality to report accounts you believe are breaking our ToS:

https://docs.github.com/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

![Screenshot 2024-06-12 at 22 31 58](https://github.com/community/community/assets/330170/c3a9213a-ee37-4504-9d05-cf68a685f74e)

**Reply by RedProGroup** [reply]
> Hi @RedProGroup 👋
> 
> Please use this functionality to report accounts you believe are breaking our ToS:
> 
> https://docs.github.com/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam
> 
> ![Screenshot 2024-06-12 at 22 31 58](https://private-user-images.githubusercontent.com/330170/339138252-c3a9213a-ee37-4504-9d05-cf68a685f74e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyMzMwNDMsIm5iZiI6MTcxODIzMjc0MywicGF0aCI6Ii8zMzAxNzAvMzM5MTM4MjUyLWMzYTkyMTNhLWVlMzctNDUwNC05ZDA1LWNmNjhhNjg1Zjc0ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMlQyMjUyMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYmVmYTQ0ZjUyN2EzMTg3YjliM2JlYmZmODBjMzUzYzkwNTdhYjczMGUyZjg5N2JjMjFmN2YzMDFjM2Y1ZDY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.cHD9Lbcpb4GeVi7yZuI_5GmtyaPSFCaPAXQcPT2R0uQ)

Thanks for the answer!
